README.md

```diff
@@ -104,10 +104,30 @@ supabase
 ```
 
 ## Realtime Changes
-```
+```python
```

Comment:
thanks for these

Reply:
Yup! Thanks for the syntax highlighting.